templates/legal/terms-and-policies/livepatch-terms-of-service.html

```diff
@@ -2,125 +2,72 @@
 
-
 {% block content %}
-<div class="p-strip">
+
+<div class="p-strip is-deep">
   <div class="row">
     <div class="col-8">
-
       <h1 id="title">Livepatch Terms of Service</h1>
-
       <h2 id="intro">1. Introduction</h2>
-
-      <p>These Terms of Service cover the provision of Canonical&#39;s Livepatch Service (the "<strong>Service</strong>") for the Livepatch software ("<strong>Software</strong>") by Canonical Group Limited ("<strong>Canonical</strong>", "<strong>us</strong>",
-        "<strong>our</strong>" or "<strong>we</strong>") to you ("<strong>you</strong>" or "<strong>your</strong>") subject to and in accordance with these Terms of Service. Please read these Terms of Service carefully before you use the Service. By using
-        the Service, you agree to become bound by these Terms of Service.</p>
-
+      <p>These Terms of Service cover the provision of Canonical&#39;s Livepatch Service (the "<strong>Service</strong>") for the Livepatch software ("<strong>Software</strong>") by Canonical Group Limited ("<strong>Canonical</strong>", "<strong>us</strong>","<strong>our</strong>" or "<strong>we</strong>") to you ("<strong>you</strong>" or "<strong>your</strong>") subject to and in accordance with these Terms of Service. Please read these Terms of Service carefully before you use the Service. By using the Service, you agree to become bound by these Terms of Service.</p>
       <p>If you have an executed agreement with a Canonical group company which includes a licence to the Software your use of the Software is governed by that agreement. Otherwise, these Terms of Service and the Livepatch licence will apply.</p>
-
-      <p>If you are entering into these Terms of Service on behalf of a company or legal entity, you represent that you have authority to bind such company or legal entity, its officers, employees and agents and all users who access the Service through the
-        account, to these Terms of Service. "you" and "your" shall refer to the company or legal entity and such users. If you do not have such authority do not accept these Terms of Service.</p>
-
+      <p>If you are entering into these Terms of Service on behalf of a company or legal entity, you represent that you have authority to bind such company or legal entity, its officers, employees and agents and all users who access the Service through the account, to these Terms of Service. "you" and "your" shall refer to the company or legal entity and such users. If you do not have such authority do not accept these Terms of Service.</p>
       <p>You must be at least 13 years old to use the Service. If you are between age 13 and 18, you confirm that you have your parent&#39;s or legal guardian&#39;s consent to use the Services and that they have read and agreed to these Terms of Service.</p>
-
       <p>Canonical may update these Terms of Service from time to time in its discretion.</p>
-
       <section>
         <h2 id="service">2. Service</h2>
```

Comment:
@pmahnke The whole container is wrapped in col-8? Do you think that it shouldn't be?